.github/workflows/release.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         uses: docker/login-action@v1
         with:
           registry: ghcr.io
-          username: ${{ github.repository_owner }}
+          username: ${{ sccrets.DOCKER_USER }}
```

Comment:
How about using `CR_USER` and `CR_PAT` so we won't be confused?

For the better `GHCR_USERNAME` & `GHCR_TOKEN`.

Reply:
SGTM